transaction/atomic.go

```diff
@@ -54,11 +54,8 @@ func getAccountDetails(hezClient client.HermezClient, address string,
 	return
 }
 
-// AtomicTransfer perform token or ETH transfers in a pool of transactions
-func AtomicTransfer(hezClient client.HermezClient, ethereumChainID int,
-	txs []AtomicTxItem) (serverResponse string, err error) {
-	atomicGroup := api.AtomicGroup{}
-
+// CreateFullTxs turn the basic information in a PoolL2Tx
+func CreateFullTxs(hezClient client.HermezClient, txs []AtomicTxItem) (fullTxs []hezcommon.PoolL2Tx, err error) {
```

Comment:
This is not art of this PR, ut something I've realized by looking at the function code:

Instead of

```go
localTx.Type = hezcommon.TxTypeTransfer
// ...
err = localTx.SetID()
```

Use

```go
hezcommon.NewPoolL2Tx(localTx)
```

This function set the id and type correctly (right now type it's always transfer, which is used to indicate transfer to idx actually, but it could be transfer to eth addr, transfer to bjj, exit, ...)